zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -255,3 +269,49 @@ def get_parse_product_request(
         )
         scrapy_request.meta["allow_offsite"] = True
         return scrapy_request
+
+
+class ExperimentalEcommerceSpiderParams(BaseModel):
+    urls: List[str] = Field(
+        title="URLs",
+        description=(
+            "Initial URLs for the crawl, separated by new lines. Enter the "
+            "full URL including http(s), you can copy and paste it from your "
+            "browser. Example: https://toscrape.com/"
+        ),
+    )
+    crawl_strategy: EcommerceCrawlStrategy = CRAWL_STRATEGY_FIELD
+    geolocation: Optional[Geolocation] = GEOLOCATION_FIELD
+    max_requests: Optional[int] = MAX_REQUESTS_FIELD
+    extract_from: Optional[ExtractFrom] = EXTRACT_FROM_FIELD
+
+    @field_validator("urls", mode="before")
+    @classmethod
+    def split_lines(cls, value: Union[List[str], str]) -> List[str]:
+        if isinstance(value, str):
+            value = [v.strip() for v in value.split("\n")]
```

Comment:
@BurnzZ what do you think about checking URLs pattern for each line also here?

Reply:
https://github.com/zytedata/zyte-spider-templates/pull/38/commits/3f39de4d74ce8e711530d168bd34caace2449de0